crates/cheatcodes/spec/src/vm.rs

```diff
@@ -2313,15 +2313,18 @@ interface Vm {
     function ensNamehash(string calldata name) external pure returns (bytes32);
 
     /// Returns a random uint256 value.
-    #[cheatcode(group = Utilities)]
+    /// `randomUint` is being deprecated in favor of `arbitraryUint`. It will be removed in future versions.
+    #[cheatcode(group = Utilities, status = Deprecated(Some("replaced by `arbitraryUint`")))]
     function randomUint() external returns (uint256);
 
     /// Returns random uin256 value between the provided range (=min..=max).
-    #[cheatcode(group = Utilities)]
+    /// `randomUint` is being deprecated in favor of `arbitraryUint`. It will be removed in future versions.
+    #[cheatcode(group = Utilities, status = Deprecated(Some("replaced by `arbitraryUint`")))]
     function randomUint(uint256 min, uint256 max) external returns (uint256);
 
     /// Returns a random `address`.
-    #[cheatcode(group = Utilities)]
+    /// `randomAddress` is being deprecated in favor of `arbitraryAddress`. It will be removed in future versions
+    #[cheatcode(group = Utilities, status = Deprecated(Some("replaced by `arbitraryAddress`")))]
     function randomAddress() external returns (address);
```

Comment:
Yeah, there were some discussions re how to name these, per @ehildenb
`random would be a bit weird to use for symbolic execution, fresh or abitrary would be better`
 and we already added `setArbitraryStorage` instead `setRandomStorage` (see https://github.com/foundry-rs/foundry/pull/8807#discussion_r1747011093)

Reply:
These are not specific to symbolic execution

Reply:
Agree, @ehildenb is ok to stick with random per Dani point?

Reply:
renamed from arbitrary to random in https://github.com/foundry-rs/foundry/pull/8882/commits/6820df674644123359ccd7b53d6d9af0dc5b2343, we can discuss further namings but should be OK to stick with random for now

Reply:
Sorry for the delayed reply. I still would prefer `fresh` or `arbitrary`. The point of this change is to allow Foundry property tests to either be interpreted _existentially_ (with random sampling), *or* _universally_ (with symbolic execution). So I guess to me, the point is that `fresh` and `arbitrary` are neither specific to fuzzing nor symbolic execution (and work for both), and that's a bonus because it means the same test can be interpreted in both ways.

But we'll be fine with `random`, it just means the tests will read a bit funny for symbolic execution, where we'll have to tell users "random is interpreted differently here, as a fresh logical variable".